ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-nightly-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -262,6 +274,63 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-aws-upi
+- as: azure-ipi-disc-fullyprivate-f28
```

Comment:
I did not see any azure ci jobs in openshift-openshift-tests-private-release-4.11__arm64-nightly.yaml

Reply:
openshift-openshift-tests-private-release-4.12__arm64-stable.yaml has this profiles, I was copied all jobs to both y and z stream